generated/nidigital/nidigital/enums.py

```diff
@@ -99,7 +195,13 @@ class SequencerRegister(Enum):
 
 class _SiteResultType(Enum):
     PASS_FAIL = 3300
+    r'''
+    Pass/fail site result.
+    '''
     CAPTURE_WAVEFORM = 3301
+    r'''
+    Capture waveform site result.
+    '''
 
 
 class SoftwareTrigger(Enum):
```

Comment:
Documentation missing.

Could use the strings from https://github.com/ni/nimi-python/blob/820327d176ac6397cb361c38e4df11deaa669b76/src/nidigital/metadata/functions.py#L2445

Reply:
Done.